social_core/backends/mediawiki.py

```diff
@@ -0,0 +1,175 @@
+"""
+MediaWiki OAuth1 backend
+========================
+
+Usage
+-----
+
+In addition to the general setup you need to define the
+following parameters. In Django's settings.py you would
+use for English Wikipedia::
+
+    SOCIAL_AUTH_MEDIAWIKI_KEY = <consumer_key>
+    SOCIAL_AUTH_MEDIAWIKI_SECRET = <consumer_secret>
+    REQUEST_TOKEN_URL =
+        'https://en.wikipedia.org/w/index.php?title=Special:OAuth/initiate'
+    AUTHORIZATION_URL =
+        'https://en.wikipedia.org/w/index.php?title=Special:Oauth/authorize'
+    ACCESS_TOKEN_URL =
+        'https://en.wikipedia.org/w/index.php?title=Special:Oauth/token'
+    IDENTIFY_URL =
+        'https://en.wikipedia.org/w/index.php'
+
+If users should only be prompeted once for permission use:
```

Comment:
Good idea to document how the grant system works.

Will only store the base URL in the settings.